workflows/spark-bench/ansible/playbook.yml

```diff
@@ -0,0 +1,33 @@
+- hosts: master
+  tasks:
+  - name: run spark master
+    shell: |
+      docker run --name spark-master
+      --net=host
+      -h spark-master
+      -e ENABLE_INIT_DAEMON=false
+      -d
+      popperized/spark-master
+
+- hosts: workers
+  tasks:
+  - name: run spark worker
+    shell: |
+      docker run --name spark-worker
+      --net=host
+      --link spark-master:spark-master
```

Comment:
this is not needed, as this only works for when the containers run on the same machine